container/features/src/main/resources/features.xml

```diff
@@ -662,6 +662,12 @@
       <bundle>mvn:org.opennms.core.ipc.rpc/org.opennms.core.ipc.rpc.aws-sqs-impl/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-core-ipc-rpc-kafka" description="OpenNMS :: Core :: IPC :: RPC :: Kafka." version="${project.version}">
```

Comment:
Nit: Name should be `OpenNMS :: Core :: IPC :: RPC :: Kafka Impl.` to follow the naming convention of the other IPC implementations.